### ISSUE TITLE
Add code and highlighting for operations in OrderedArray.py

### ISSUE THREAD

**Issue by JMCanning78** [JMCanning78/datastructures-visualization#98]
Display the code for each of the operations in the OrderedArray visualization and highlight the active code.  Display the code for the New, Search, Insert, and Delete operations.  For short, O(1), operations, make a call to `wait(0.2)` for each change in highlight so users will have some time to see it. 

Here's the textbook code for the ordered array:
```
class OrderedArray(object):
   def __init__(self, initialSize):    # Constructor
      self.__a = [None] * initialSize  # The array stored as a list
      self.__nItems = 0                # No items in array initially

   def get(self, n):                   # Return the value at index n
      if 0 <= n and n < self.__nItems: # Check if n is in bounds, and
         return self.__a[n]            # only return item if in bounds
      raise IndexError("Index " + str(n) + " is out of range")
         
   def find(self, item):            # Find index at or just below
      lo = 0                        # item in ordered list
      hi = self.__nItems-1          # Look between lo and hi
      
      while lo <= hi:
         mid = (lo + hi) // 2       # Select the midpoint
         if self.__a[mid] == item:  # Did we find it at midpoint?
            return mid              # Return location of item
         elif self.__a[mid] < item: # Is item in upper half?
            lo = mid + 1            # Yes, raise the lo boundary
         else: 
            hi = mid - 1            # No, but could be in lower half
            
      return lo   # Item not found, return insertion point instead   

   def search(self, item):
      index = self.find(item)       # Search for item
      if index < self.__nItems and self.__a[index] == item:
         return self.__a[index]     # and return item if found
   
   def insert(self, item):    # Insert item into the correct position
      if self.__nItems >= len(self.__a): # If array is full,
         raise Exception("Array overflow") # raise exception
      
      j = self.__nItems       # Start at right (larger) end and loop
      while 0 < j and self.__a[j - 1] > item: # over items
         self.__a[j] = self.__a[j-1] # Move larger item to right
         j -= 1               # Advance left among items
         
      self.__a[j] = item      # Insert the item
      self.__nItems += 1      # Increment the number of items

   def delete(self, item):             # Delete any occurrence
      j = self.find(item)              # Try to find the item
      if j < self.__nItems and self.__a[j] == item:  # If found,
         self.__nItems -= 1            # One fewer at end
         for k in range(j, self.__nItems): # Move bigger items left
            self.__a[k] = self.__a[k+1]
         return True                   # Return success flag

      return False            # Made it here; item not found

   def traverse(self, function=print): # Traverse all items
      for j in range(self.__nItems):   # and apply a function
         function(self.__a[j])
```

**Comment by ShaniLewis** [comment]
we noticed Ordered Array has a active issue requesting that find be changed to be search and we wanted to know if you could merge that change into the master so that we can add the correct code highlighting for it.

**Comment by JMCanning78** [comment]
It won't hurt to change the button's label from "Find" to "Search" in separate Issues and PRs.  I don't plan to merge any PRs until they are approved by at least one reviewer and I've taken at least a short look at them.  I'm not sure if you were asking me to merge something early or not, or just asking to make the change in responding to different issues.

**Comment by JMCanning78** [comment]
The code highlighting API changes in PR #159.  Also this issue is likely to be replaced by a separate issue to convert OrderedArray to be a subclass of `SortingBase`.  That will standardize the Insert, Search, Delete, Traverse, and maybe a couple of other operations.  The `OrderedArray` class will need its own version of the `find()` method that performs binary search rather than linear search.

**Comment by JMCanning78** [comment]
This issue has been replaced by Issue #161 and is being addressed in PR #159.
